lean4-infoview/src/infoview/index.css

```diff
@@ -118,3 +121,17 @@ select {
 .tooltip[data-popper-placement^='right'] > .tooltip-arrow {
     left: -4px;
 }
+
+.popup-menu-text { text-align: center; padding: 5px; vertical-align:top; display:inline-block; }
```

Comment:
I missed this before, but so far we have mostly been using [Tachyons](http://tachyons.io/docs/table-of-styles/) rather than bespoke CSS classes for styling of this kind (with the exception of this exact file, of course). So instead of `popup-menu-text` one would write `tc pa2 v-top dib` as the `className` (`pa1` or `pa2`, Tachyons intentionally doesn't provide specific pixel offsets for padding so as to keep a consistent, fixed scale). All the other classes either use `var` or *need* specific offsets to match a font, so it only applies to `popup-menu-text`. I will leave to you whether you want to change this, it's not the end of the world if we use classes.

Reply:
Interesting, I have lots of questions, so I moved this to a new issue  https://github.com/leanprover/vscode-lean4/issues/183